File: 485-max-consecutive-ones/max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        c=0
        ans=0
        for i in range(len(nums)):
            if nums[i]==1:
                ans +=1
            else:
                ans = 0
            c = max(ans,c)
        return c

File: 485-max-consecutive-ones/max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int n = nums.size();
        int ans=0,c=0;
    
        for(int i=0;i<n;i++){
            if(nums[i])
            ans++;
            else 
            ans=0;
            c=max(ans,c);
        }
        return c;
        
    }
};